lib/chef/provisioning/aws_driver/driver.rb

```diff
@@ -675,31 +675,42 @@ def aws_credentials
                            end
     end
 
-    def default_ami_for_region(region)
-      Chef::Log.debug("Choosing default AMI for region '#{region}'")
+    def default_ami_arch
+      'amd64'
+    end
 
-      case region
-        when 'ap-northeast-1'
-          'ami-6cbca76d'
-        when 'ap-southeast-1'
-          'ami-04c6ec56'
-        when 'ap-southeast-2'
-          'ami-c9eb9ff3'
-        when 'eu-west-1'
-          'ami-5f9e1028'
-        when 'eu-central-1'
-          'ami-56c2f14b'
-        when 'sa-east-1'
-          'ami-81f14e9c'
-        when 'us-east-1'
-          'ami-12793a7a'
-        when 'us-west-1'
-          'ami-6ebca42b'
-        when 'us-west-2'
-          'ami-b9471c89'
-        else
-          raise 'Unsupported region!'
+    def default_ami_release
+      'trusty'
+    end
+
+    def default_ami_root_store
+      'ebs'
+    end
+
+    def default_ami_virtualization_type
+      'paravirtual'
```

Comment:
I think both are fine choices as defaults. New instance types seem to be HVM only as are the burstable instances people might use more with the free tier.